performance-timeline/po-mark-measure.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>PerformanceObservers: mark and measure</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="performanceobservers.js"></script>
+<h1>PerformanceObservers: mark and measure</h1>
+<p>
+Performance.mark() and Performance.measure() will <a href="https://w3c.github.io/performance-timeline/#dfn-queue-a-performanceentry">queue a PerformanceEntry</a>.
+</p>
+<div id="log"></div>
+<script>
+  async_test(function (t) {
+    var stored_entries = [];
+    var observer  = new PerformanceObserver(
+        t.step_func(function (entryList, obs) {
+          stored_entries =
+            stored_entries.concat(entryList.getEntries());
+          if (stored_entries.length >= 4) {
+            checkEntries(stored_entries,
+              [{ entryType: "mark", name: "mark1"},
+              { entryType: "mark", name: "mark2"},
+              { entryType: "measure", name: "measure1"},
+              { entryType: "measure", name: "measure2"}]);
+            observer.disconnect();
+            t.done();
+          }
+        })
+      );
+    observer.observe({entryTypes: ["mark", "measure"]});
+  }, "entries are observable");
+
+  async_test(function (t) {
+    var mark_entries = [];
+    var observer = new PerformanceObserver(
+        t.step_func(function (entryList, obs) {
+          mark_entries =
+            mark_entries.concat(entryList.getEntries());
+          if (mark_entries.length >= 2) {
+            checkEntries(mark_entries,
+              [{ entryType: "mark", name: "mark1"},
+              { entryType: "mark", name: "mark2"}]);
+            checkEntries(entryList.getEntriesByName("mark1"),
```

Comment:
I should update that check. Was meant to check that getEntriesByName works but then I buffered the entries and forgot to update that check.